code/modules/clothing/head/misc.dm

```diff
@@ -489,3 +489,21 @@
 	flags_inv = 0
 	armor = list(MELEE = 30, BULLET = 15, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 0, RAD = 0, FIRE = 50, ACID = 50)
 	strip_delay = 140
+
```

Comment:
Why not put this under ego_gear.dm? It is mostly relevant to ego gear anyways, and it could be difficult to find in its current state.

Reply:
Primarily because it's a hat and not directly EGO gear, I could leave a comment towards this file to point there or move it. I thought this spot fit better.